NTPClient.h

```diff
@@ -30,6 +31,7 @@ class NTPClient {
     NTPClient(UDP& udp);
     NTPClient(UDP& udp, int timeOffset);
     NTPClient(UDP& udp, const char* poolServerName);
+    NTPClient(UDP& udp, IPAddress poolServerIP);
     NTPClient(UDP& udp, const char* poolServerName, int timeOffset);
     NTPClient(UDP& udp, const char* poolServerName, int timeOffset, int updateInterval);
```

Comment:
That would certainly work. I didn't add that code because I find it strange having multiple functions all doing much the same thing - I guess I'm just not used to the way things are structured in C++ ? A single function with optional arguments would seem more 'natural' to me. e.g.,

`    NTPClient(UDP& udp, IPAddress poolServerIP, int timeOffset = -1, int updateInterval = -1);`